### PR TITLE
KiCad PCM Support

### PR DESCRIPTION
**Pull request by JarrettR**
This will require running a python script, and then creating a new release to have any effect.

Then it'll need to be submitted to KiCad - Full directions are in `release-kicad-addons.md` and I can clarify if there is any confusion

### CODE REVIEW

**Comment by JarrettR** [comment]
Certainly, I think I've done it

**Comment by pedrominatel** [comment]
The workflow will run after the release is created as a draft and the zip file will be added as an asset.
Now we will work to integrate the workflow with the KiCad GitLab to have this process fully automated.

File: footprints/Espressif.pretty/ESP32-C3-MINI-1.kicad_mod [hunk 1]
```diff
@@ -145,21 +145,7 @@
...
-      (pts
-        (xy -6.6 -2.9)
-        (xy -6.6 -8.3)
-        (xy 6.6 -8.3)
-        (xy 6.6 -2.9)
-      )
-    )
-  )
-  (model "${ESPRESSIF_3DMODELS}/ESP32-C3-MINI-1.STEP"
+  (model "${KICAD6_3RD_PARTY}/3dmodels/com_github_espressif_kicad-libraries/espressif.3dshapes/ESP32-C3-MINI-1.STEP"
```
Comment (pedrominatel) [inline footprints/Espressif.pretty/ESP32-C3-MINI-1.kicad_mod:148]: Should we consider KiCad 7 and backport to KiCad 6 on the legacy branch?

Comment (JarrettR) [reply footprints/Espressif.pretty/ESP32-C3-MINI-1.kicad_mod:148]: Certainly, this looks like a mistake I made in merging. I don't even really think it's worthwhile backporting to 6 - PCM is a convenience, it's not really critical. So 6 users are still able to use the branch as a direct download as the status quo

**Comment by pedrominatel** [comment]
Looks like we are able to merge and create the 2.0.0 version.
Thank you for this contribution!

**Pull request merged**